examples/regress_173_test.go

```diff
@@ -66,8 +66,7 @@ func TestRegress173(t *testing.T) {
   "response": {
     "changes": "DIFF_SOME",
     "diffs": [
-      "privateKeyPem",
-      "subject"
```

Comment:
Looks like this had a spurious diff on subject. With the new bridge version this diff is eliminated. This looks like a good change.